### ISSUE TITLE
Timing Adjustable and Pause, Stop, Hide criteria are in the wrong order

### ISSUE THREAD

**Issue by maggiewachs** [GSA/openacr#347]
OpenACR files are showing the following: 
2.2.1 Pause, Stop, Hide
2.2.2 Timing Adjustable

When the order should be:
2.2.1 Timing Adjustable
2.2.2 Pause, Stop, Hide

See WCAG 2.1 documentation: https://www.w3.org/TR/WCAG21/#timing-adjustable

If you do a search of "timing adjust" in this repo, you'll see this needs to be corrected in several files: https://github.com/search?q=repo%3AGSA%2Fopenacr%20timing%20adjust&type=code

**Comment by dmundra** [comment]
Bug fix in the fork. After it is merged, I will open a pull request into this repo to merge it.
